### ISSUE TITLE
Basic fluid tank from mekanism is bugged

### ISSUE THREAD

**Issue by SinisterSauce** [IamRyan30/Decursio-Team#14]
### Modpack Version

pre release 26

### Launcher Used

curseforge

### Did this issue occur on Multiplayer, Singleplayer, or both?

Both

### Do you, or did you had Optifine installed when the issue took place?

No

### Tried reproducing without Optifine?

No

### Describe your issue

can automate the basic fluid tank. I can only craft one if i place the fluid cell into the world and then switch a side to input then it will go into the recipe.

### Log File related to your issue

https://mclo.gs/DALUvJM

### Searched for known issues?

I did not search

**Comment by IamRyan30** [comment]
You can, or can't automate it? Does this persist in the 27th version? If it does, please leave the log from the client and from the server here; there might be something useful on them. And please leave them on a proper web page, like pastebin or gists.

**Comment by SinisterSauce** [comment]
I am now on 27. Same thing.

I can autocraft the fluid cell. The fluid cell is part of the basic fluid tank from mekanism. I have a recipe for the mek tank but it wont accept the fluid cell until i mess with its side configurations. Im assuming its an NBT data issue with the fluid cell.

Server Log: [https://gist.github.com/SinisterSauce/ab8d802517899766f00c03c24ee96ddf](url)

Client Log: [https://gist.github.com/SinisterSauce/bb0572192a441bab29d78bf0767795cc](url)

**Comment by IamRyan30** [comment]
Ok, I couldn't reproduce this. I tried in singleplayer only and I'll look into it more, to find out what's causing this. However. in general, this won't matter anyways. I decided to change the recipes for the Tank, and Cube, so they'll use cell frames, instead of the cells. I'll also change the frame's recipes, so they won't use the Pity Machine Frame anymore, to reduce the amount of Petcoke required overall. I know, it's not really a fix and unfortunately, this won't be available for the next few weeks, but I'll try to get this out as soon as possible. Apologies for slowing down your progress.
